apps/condo/domains/ticket/schema/TicketChange.test.js

```diff
@@ -517,6 +516,41 @@ describe('TicketChange', () => {
 
                 expect(obj.actualCreationDate).toEqual(statusUpdatedAt)
             })
+
+            it('is not filled if statusUpdatedAt is not in the payload', async () => {
+                const client = await makeClientWithNewRegisteredAndLoggedInUser()
+                const [organization] = await createTestOrganization(admin)
+                const [property] = await createTestProperty(admin, organization)
+                const [role] = await createTestOrganizationEmployeeRole(admin, organization, { canManageTickets: true })
+                await createTestOrganizationEmployee(admin, organization, client.user, role)
+
+                const [ticket] = await createTestTicket(client, organization, property)
+                const statusUpdatedAt = dayjs().subtract('20', 'second').toISOString()
+
+                // Update 1 Update status explicitly. TicketChange actualUpdateDate should equal this number
+                await updateTestTicket(client, ticket.id, {
+                    status: { connect: { id: STATUS_IDS.IN_PROGRESS } },
+                    statusUpdatedAt,
+                })
+
+                // Explicitly sleep for some time
+                await new Promise(r => setTimeout(r, 500))
```

Comment:
waitFor would be even better here

Reply:
No, since we need to sleep between requests, not wait for something